tools/render_gh_markdown

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/ruby
+#
+# Renders GitHub-flavored Markdown to HTML
+#
+# render_gh_markdown <file> ...
+#
+# Requires the github-markup gem to be installed. Rendered output is
+# written to .html files next to the original input files.
+
+# TODO: Figure out how to generate anchors. The index links in Commands.md 
+# aren't working when rendered locally with this, though they do work when
+# viewed on GitHub.
+
+require 'optparse'
+require 'github/markup'
+
+options = {}
+OptionParser.new do |opts|
+	opts.banner = "Usage: render_gh_markdown.rb <file> ..."
+	opts.on("-v", "--verbose", "Run verbosely") do |v|
+		options[:verbose] = v
+	end
+end.parse!
+
+infiles = ARGV
+
+if infiles.empty?
+	$stderr.puts("Error: no files specified")
+	abort
+end
+
+for infile in infiles do
+	basefile = infile.gsub(/\.[^.]*$/, '')
```

Comment:
If my guess about the regex is correct, this line does the same as `basefile = File.basename(infile, File.extname(infile))` ? It will remove the last part of extension.